src/disk_commands.p8

```diff
@@ -277,7 +273,7 @@ io_error:
         if_cs {
             newfilename = main.command_arguments_ptr + space_idx + 1
             main.command_arguments_ptr[space_idx] = 0
-            diskio.list_filename[0] = 'c'
+            diskio.list_filename[0] = petscii:'c'
             diskio.list_filename[1] = ':'
```

Comment:
for consistency, please also add petscii: prefix to the other 2 characters (even though the resulting byte value will be the same)

Reply:
Ok, will do, when i get back home

Reply:
@irmen